catalog-info.yaml

```diff
@@ -645,6 +645,54 @@ spec:
 # SECTION END: Benchmark pipeline
 # *******************************
 
+# ***********************************
+# SECTION START: Benchmark multiple Logstash versions
+# ***********************************
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-benchmark-versions-pipeline
```

Comment:
I don't recall default `schedules:` value. Do you know if it runs periodically?

Reply:
Right, no schedule is set here. I think we don't need to run every versions regularly, instead, it is a pipeline triggered manually for calibration on new cluster or investigation on specific versions

Reply:
Yup, the intention is we run when required. It is my flaky memory that says `schedules:` wasn't exist but CI periodically run. I have double checked the doc, no such info, we seem safe.